src/text/2-package-subsections/datasets.tex

```diff
@@ -9,3 +9,24 @@ \subsection{gammapy.datasets}
 	\caption{Using gammapy.data to access DL3 level data with a DataStore}
 	\label{fig*:minted:gp_datasets}
 \end{figure}
+
+Reduced data, IRFs, and models are bundled together to create a Data Level 4 (DL4). This is provide in the Dataset class which computes a likelihood to interface with the Fit class to perform a fitting. Corresponding to the different analysis types, five types of datasets are supported in gammapy (\ref{table:datasets_types}). 3D data cubes (with two spatial and one energy) are represented by MapDatasets, whereas SpectrumDatastes contain binned energy information for a given on region. 2D images are a special case of MapDatasets, with only one bin in energy. The FluxPointsDataset is used to fit pre-computed flux points when no convolution with IRFs are needed.   
```

Comment:
provide -> provided
SpectrumDatastes -> SpectrumDatasets
given on region: should we give here a link towards a documentation of regions?